packages/telegraf/extra/start_telegraf.sh

```diff
@@ -24,6 +24,14 @@ mkdir -p "${TELEGRAF_CONTAINERS_DIR}"
 # Migrate old containers dir to new location in case the cluster was upgraded.
 /opt/mesosphere/active/telegraf/tools/migrate_containers_dir.sh "${LEGACY_CONTAINERS_DIR}" "${TELEGRAF_CONTAINERS_DIR}"
 
+# Add symlinks to any additional configuration files in the user-editable Telegraf config directory
+shopt -s nullglob
+for file in ${TELEGRAF_EXTRA_CONFIG_DIR}*; do
+	ln -sfn $file ${TELEGRAF_CONFIG_DIR}$(basename "$file")
```

Comment:
If a user adds a configuration file and then removes it, what happens? Does it just leave behind a broken symlink, or does it cause problems?

Reply:
Broken symlinks are removed from the telegraf config dir [below](/var/lib/dcos/telegraf/telegraf.d)!

Reply:
how did I miss this 🤦‍♂️